packages/js/src/executors/release-publish/extract-npm-publish-json-data.ts

```diff
@@ -0,0 +1,62 @@
+const expectedNpmPublishJsonKeys = [
+  'id',
+  'name',
+  'version',
+  'size',
+  'filename',
```

Comment:
To confirm, ALL of these keys are present in EVERY version of NPM?

Reply:
We don’t support anywhere close to every version of npm anywhere else in the codebase 😅 I will manually verify against all versions we do support

Reply:
Manually tested all the way back to npm v7 (which is from before the range of node that we support), we are all good